pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -7573,6 +7575,23 @@ func TestChangefeedBackfillCheckpoint(t *testing.T) {
 		// Resume job.
 		require.NoError(t, jobFeed.Resume())
 
+		// Verify that the resumed job has restored the progress from the checkpoint
+		// to the change frontier.
+		timestampSpansMap := checkpoint.ConvertLegacyCheckpoint(jobCheckpoint, hlc.Timestamp{}, hlc.Timestamp{})
+		expectedFrontier, err := resolvedspan.NewCoordinatorFrontier(jobCheckpoint.Timestamp, *h, tableSpan)
+		if err != nil {
+			t.Fatal(err)
+		}
+		assert.NoError(t, checkpoint.Restore(expectedFrontier, timestampSpansMap))
+
+		checkedCoordinatorFrontier := atomic.Bool{}
```

Comment:
nit: `var checkedCoordinatorFrontier atomic.Bool`

there's no need to manually assign a zero value